cmd/utils/spinner.go

```diff
@@ -105,3 +106,11 @@ func ucFirst(str string) string {
 	}
 	return ""
 }
+
+func StopSpinnerIfInterruptSignal(spinner *Spinner) {
+	stopChannel := make(chan os.Signal, 1)
```

Comment:
Let's move this logic to the main goroutine, and wait for CTRL+C or the end of the execution in the main goroutine.